scripts/package-harvester-os

```diff
@@ -78,7 +78,7 @@ then
 fi
 
 INITRD=$(docker run --rm ${HARVESTER_OS_IMAGE} readlink /boot/initrd)
-docker create --cidfile=os-img-container ${HARVESTER_OS_IMAGE}
+docker create --cidfile=os-img-container ${HARVESTER_OS_IMAGE} -- sleep 30
```

Comment:
Why do we need this? Is this a requirement of a newer version of Docker? (Also, you could maybe try something like `tail -f /dev/null` which should keep it running forever until it's explicitly removed - I never really liked fixed sleeps :-) )

Reply:
Thanks @tserong.
Yeah, the newer version Docker requires the entry point (means the command to execute).
Thanks for the hint. The `tail -f /dev/null` is the better way. Updated!